src/index/mod.rs

```diff
@@ -228,10 +242,23 @@ pub struct IndexView {
 }
 
 impl IndexView {
-    #[allow(dead_code)]
     pub fn len(&self) -> u32 {
+        self.sealed_len() + self.growing_len() + self.write_len()
```

Comment:
These functions should be merged too, otherwise inconsistency data might be seen.

Reply:
I think we should just send information as many as possible.

Reply:
https://github.com/silver-ymz/pgvecto.rs/blob/index-progress/src/bgworker/worker.rs#L132-L152

The functions are all called from the same view, the data will always be consistent.

Reply:
`write_len` may change.

Reply:
> write_len may change.

write is an `Arc` in `IndexView`. It won't be changed. 

In worker.rs:133, `ArcSwap::load_full` will make a copy for view arc.

Reply:
updated in new commit